### PR TITLE
Update Slack Import Limitations

### PR DESCRIPTION
**Pull request by yangchen1**

### CODE REVIEW

**Comment by esethna** [comment]
+1

File: source/help/settings/team-settings.md [hunk 1]
```diff
@@ -50,6 +50,6 @@ The Slack Import feature in Mattermost is in "Beta" and focus is on supporting m
 #### Limitations 
 
 - Newly added markdown support in Slack's Posts 2.0 feature announced on September 28, 2015 is not yet supported.
-- Slack does not export files or images your team has stored in Slack's database. Mattermost will provide links to the location of your assets in Slack's web UI.
+- Slack does not export files or images your team has stored in Slack's database. Mattermost will provide links to the location of your assets in Slack's web UI. However, you can use [this tool](https://github.com/grundleborg/slack-advanced-exporter) to include files in the Slack export, and import them into Mattermost.
 - Slack does not export any content from private groups or direct messages that your team has stored in Slack's database. 
-- In Beta, Slack accounts with usernames or email addresses identical to existing Mattermost accounts will not import. No pre-check or roll-back is currently offered. 
+- In Beta, Slack accounts with usernames or email addresses identical to existing Mattermost accounts will be merged. No pre-check or roll-back is currently offered.
```
Comment (lfbrock) [inline source/help/settings/team-settings.md]: Do you know if LDAP or SSO accounts are merged, or is it only if the existing account has email login and a username + email address that match the Slack import?

Comment (yangchen1) [inline source/help/settings/team-settings.md]: The latter. :) Should I add: "...Mattermost accounts, including ones using LDAP/SSO for login,.."

**Comment by lfbrock** [comment]
+1

**Pull request merged**